### DISCUSSION TITLE
Cannot download media from private chat OnlyF

### DISCUSSION THREAD

**Discussion by ojiegik25** [AAndyProgram/SCrawler#167] [General]
Hi and thanks for the great program!
Configured the program to work with onlyf, the posts download perfectly, but can not download purchased videos in private chat.
Attribute with a video named "nodownload"

![image](https://github.com/AAndyProgram/SCrawler/assets/24476355/93038d0c-515b-4696-bcc9-2820aec0ac32)

Here's what the Network tab looks like
![image](https://github.com/AAndyProgram/SCrawler/assets/24476355/8f32a033-9c43-4ea1-a48e-fd4cc3571ea6)

This video has DRM protection. Is it possible to download such videos? 
I don't see anything wrong with keeping purchased videos in a personal archive.

**Comment by AAndyProgram** [comment]
> This video has DRM protection. Is it possible to download such videos?

No
